scout/utils/scout_requests.py

```diff
@@ -28,30 +30,24 @@ def get_request(url):
     """
     try:
         LOG.info("Requesting %s", url)
-        response = urllib.request.urlopen(url, timeout=20)
-        if url.endswith(".gz"):
-            LOG.info("Decompress zipped file")
-            data = gzip.decompress(response.read())  # a `bytes` object
-        else:
-            data = response.read()  # a `bytes` object
-        decoded_data = data.decode("utf-8")
-    except HTTPError as err:
+        response = requests.get(url, timeout=20)
+        if response.status_code != 200:
+            response.raise_for_status()
+        LOG.info("Encoded to %s", response.encoding)
+    except requests.exceptions.HTTPError as err:
         LOG.warning("Something went wrong, perhaps the api key is not valid?")
         raise err
-    except URLError as err:
-        LOG.warning("Something went wrong, are you connected to internet?")
+    except requests.exceptions.MissingSchema as err:
+        LOG.warning("Something went wrong, perhaps url is invalid?")
         raise err
-    except timeout:
+    except requests.exceptions.Timeout as err:
         LOG.error("socket timed out - URL %s", url)
-        raise ValueError
-
-    if "Error" in decoded_data:
-        raise URLError("Seems like url {} does not exist".format(url))
+        raise err
 
-    return decoded_data
+    return response
```

Comment:
Perhaps instantiate response=None at the beginning of the method?

Reply:
It will exists. Either we create it in the try, except or we raise a exception